ubuntudesign/vanilla-framework
# Add mobile nav

# Done
- Added the mobile navigation burger functionality
- Made base font size 16 for all view ports

## QA
- Pull down and run ```gulp build```
- Then run ```google-chorme demo/index.html```
- Check the navigation works on all screens sizes
- Check font stays at 16px on all view ports
